<launch>
  <arg name="status_topic" value="status"/>
  <arg name="drive_topic" value="drive"/>

  <arg name="max_steering_angle" value="0.3"/>
  <arg name="max_speed" value="5"/>

  <include file="$(find ds4_driver)/launch/ds4_driver.launch" />

  <node name="ds4_controller" pkg="ds4_controller" type="ds4_controller_node" output="screen">
    <param name="status_topic" value="$(arg status_topic)" />
    <param name="drive_topic" value="$(arg drive_topic)" />

    <param name="max_steering_angle" value="$(arg max_steering_angle)" />
    <param name="max_speed" value="$(arg max_speed)" />
  </node>
</launch>